src/services/company.filing.history.service.ts

```diff
@@ -40,5 +41,8 @@ const getCompanyFilingHistory = async (companyNumber: string, category: string,
 };
 
 const isGaz1 = (element: FilingHistoryItem) => {
-  return element.type === GAZ1_TYPE;
+  if (element.type) {
+    return element.type.toUpperCase() === GAZ1_TYPE || element.type.toUpperCase() === GAZ1A_TYPE;
+  }
+  return false;
```

Comment:
Presume this line is handling the 'undefined' scenario. Do you have a unit test for it?